webapp/src/app/shared/phaser/entities/cc-entity.ts

```diff
@@ -95,7 +107,7 @@ export abstract class CCEntity extends BaseObject {
 		pivot: Point;
 	} = <any>{};
 	
-	protected constructor(scene: Phaser.Scene, map: CCMap, x: number, y: number, typeName: string) {
+	public constructor(scene: Phaser.Scene, map: CCMap, x: number, y: number, typeName: string) {
```

Comment:
Why was this changed? Seeing the commet of `getEntity` this might be a leftover?